recipes/biobox/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "biobox" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Degiacomi-Lab/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 749f5a546b4c6e17c369bf487c7ef08e5970f9da6c2f1b790a8e6efbb4804e85
+
+build:
+  number: 0
+  script: |
+        {{ PYTHON }} setup.py build_ext --inplace
+        {{ PYTHON }} setup.py install
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - cython
+    - python
+    - setuptools
+    - numpy
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - pandas
+    - scipy
+    - scikit-learn
+test:
+  imports:
+    - biobox
+  source_files:
+    - test/
+  commands: |
+    cd test
+    python test_biobox.py
+
+about:
+  home: https://github.com/Degiacomi-Lab/biobox
+  license_file: LICENSE
+  license: GPL-3.0-only
+  summary: Biobox provides tools for loading, manipulating, and analysing atomistic and pseudoatomistic structures
+  description: |
+    Biobox provides a collection of data structures and methods for loading,
+    manipulating, and analyzing atomistic and pseudoatomistic structures.
+  doc_url: https://degiacomi-lab.github.io/biobox
+  dev_url: https://github.com/Degiacomi-Lab/biobox
+
+extra:
+    recipe-maintainers:
+        - SCMusson
+        - degiacom
+        - dclw29
```

Comment:
@degiocom and @dclw29 need to confirm in this PR that they are happy to be listed as maintainers

Reply:
I confirm I am happy to be listed as maintainer

Reply:
I confirm I am also happy to be listed as a maintainer